ITR/interfaces.py

```diff
@@ -97,11 +101,51 @@ class Config:
         validate_assignment = True
         schema_extra = schema_extra
         json_encoders = {
-            pint.Quantity: str,
+            Quantity: str,
         }
 # end of borrowing
 
 
+def emissions_quantity(dimensionality: str) -> type:
```

Comment:
`interfaces.py` is a collection of classes.
Shouldn't we avoid methods w/o classes in such situations?
it's even a method w/o a class, which has a dcorator `@classmethod`. Are you sure it's a good idea?

Reply:
The function *generates* a type, much as a class definition generates a type.

Reply:
why can't it be a method of some appropriate class within `interfaces.py`?

Reply:
Rewritten/resolved.